pandas/core/frame.py

```diff
@@ -5560,7 +5560,7 @@ def sort_index(
         self,
         axis: Axis = 0,
         level: Optional[Level] = None,
-        ascending: bool = True,
+        ascending: Union[Union[bool, int], Sequence[Union[bool, int]]] = True,
```

Comment:
also could change docstring `bool or list of bools` -> `bool or list-like of bools`

Reply:
Done.